### PR TITLE
Add basic auto complete with a feature toggle, will do a group limit on the column.

### PR DESCRIPTION
**Pull request by olymk2**

### CODE REVIEW

File: sqlite-mode-extras.el [hunk 1]
```diff
@@ -57,6 +57,9 @@
 (require 'sqlite-mode)
 (require 'view)
 
+(defvar sqlite-mode-extras-auto-complete-enabled t
```
Comment (xenodium) [inline sqlite-mode-extras.el]: Since intended for user customization, we should use defcustom.

**Review by xenodium — APPROVED** [review]
Thanks for the PR!

There were a few `M-x byte-compile-file` and `M-x checkdoc` warnings. Mind fixing those too?

File: sqlite-mode-extras.el (hunk 1 above)
Comment (olymk2) [reply sqlite-mode-extras.el]: PR should be updated, I don't write a lot of elisp so thanks for the hints.

**Review by xenodium — APPROVED** [review]
Thank you for the changes! Merging.

**Pull request merged**